pycript/Reqcheck.py

```diff
@@ -189,55 +189,63 @@ def decrypt_and_update_parameter_keys_and_values(extender, currentreq, decryptio
 def encrypt_and_update_parameters(extender, currentreq, encryptionpath, selected_method, selectedlang, body, parameters, header,selected_request_inc_ex_ctype,listofparam):
     for param in parameters:
         if selected_method == "GET" and param.getType() == IParameter.PARAM_URL:
-            encrypted_param_value = Parameterencrypt(selectedlang, encryptionpath, param.getValue())
+            parameter_value = param
+            encrypted_param_value = update_raw_value(parameter_value, selectedlang, encryptionpath, Parameterencrypt,selected_request_inc_ex_ctype, listofparam)
             currentreq = extender.helpers.updateParameter(currentreq, extender.helpers.buildParameter(param.getName(), encrypted_param_value, param.getType()))
-
         elif selected_method == "BODY" and param.getType() != IParameter.PARAM_URL:
             if param.getType() == IParameter.PARAM_BODY:
-                encrypted_param_value = Parameterencrypt(selectedlang, encryptionpath, param.getValue())
+                parameter_value = param
```

Comment:
`update_raw_value` and `update_raw_key_value` both take `Parameterencrypt` function as arg?

`Parameterencrypt` is indeed a separate function passing a func to another func reduce the executive speed at certain level due to nested conditions, 

both update function can directly call the `Parameterencrypt` function, arg is not required